internal/pkg/cli/svc_package.go

```diff
@@ -306,29 +293,32 @@ func (o *packageSvcOpts) configureClients() error {
 }
 
 type wkldCfnTemplates struct {
```

Comment:
should we just rename this type as well? it's pretty generic to work outside of workloads
```suggestion
type cfnStackConfig struct {
```